webapp/portlet/src/main/webapp/vue-apps/version-history-drawer/components/VersionCard.vue

```diff
@@ -0,0 +1,223 @@
+        <v-tooltip
+          v-if="!isRestoringVersion"
+          bottom>
+          <template #activator="{ on, attrs }">
+            <v-icon
+              v-bind="attrs"
+              v-on="on"
+              size="22"
+              class="primary--text clickable pa-0 mt-1"
+              @click="restoreVersion">
+              mdi-restart
+            </v-icon>
+          </template>
+          <span class="caption">{{ $t('versionHistory.label.restore') }}</span>
+        </v-tooltip>
+      </div>
+    </div>
+  </v-main>
+</template>
+
+<script>
+export default {
+  props: {
+    version: {
+      type: Object,
+      default: () => {
+        return {};
+      }
+    },
+    enableEditDescription: {
+      type: Boolean,
+      default: () => {
+        return false;
+      },
+    },
+    disableRestoreVersion: {
+      type: Boolean,
+      default: () => {
+        return false;
+      },
+    },
+    canManage: {
+      type: Boolean,
+      default: () => {
+        return false;
+      }
+    }
+  },
+  data: () => ({
+    dateTimeFormat: {
+      year: 'numeric',
+      month: 'long',
+      day: 'numeric',
+      hour: '2-digit',
+      minute: '2-digit'
+    },
+    MAX_DESCRIPTION_LENGTH: 500,
+    descriptionInputHidden: true,
+    newDescription: '',
+    versionObject: {},
+    isUpdatingDescription: false,
+    isRestoringVersion: false
+  }),
+  created() {
+    this.$root.$on('version-restored', (restoredVersion) => {
+      this.versionObject.current = this.versionObject.id === restoredVersion.id;
+      this.isRestoringVersion = false;
+    });
+    this.$root.$on('version-restore-error', () => {
+      this.isRestoringVersion = false;
+    });
+    this.$root.$on('version-description-updated', (version) => {
+      if (version.id === this.version.id) {
+        this.versionObject.summary = version.summary;
+        this.isUpdatingDescription = false;
+        this.descriptionInputHidden = true;
+      }
+    });
+    this.$root.$on('version-description-update-error', (version) => {
+      if (version.id === this.version.id) {
+        this.isUpdatingDescription = false;
+        this.descriptionInputHidden = false;
+      }
+    });
+    this.versionObject = Object.assign({}, this.version);
+  },
+  computed: {
+    versionDate() {
+      return this.versionObject && this.versionObject.updatedDate
+                                && this.versionObject.updatedDate.time
+                                || this.versionObject.createdDate.time;
+    },
+    descriptionMaxLengthReached() {
+      return this.newDescription && this.newDescription.length > this.MAX_DESCRIPTION_LENGTH;
+    },
+  },
+  methods: {
+    resetInput() {
+      this.descriptionInputHidden =  true;
+    },
+    updateDescription() {
+      if (this.descriptionMaxLengthReached) {
+        return;
+      }
+      this.isUpdatingDescription = true;
+      this.$emit('version-update-description', this.versionObject, this.newDescription);
```

Comment:
```suggestion
      this.$emit('version-description-updated', this.versionObject, this.newDescription);
```
Event name instead of verbs used for methods.

Reply:
this is an event to execute the action of update description, it's like the @click event it's not @clicked

Reply:
> this is an event to execute the action of update description

ok I understand